src/cpu/x64/jit_avx2_1x1_convolution.cpp

```diff
@@ -856,10 +857,7 @@ void jit_avx2_1x1_convolution_bwd_weights_t::execute_backward_weights(
         assert(IMPLICATION(!is_ddst_layout_nxc, jcp.ngroups == 1));
         const int padded_stride = utils::rnd_up(jcp.oc, jcp.oc_block);
         const int stride = jcp.oc_without_padding;
-        for (int g = 0; g < jcp.ngroups; ++g) {
-            utils::array_copy(diff_bias_in + g * stride,
```

Comment:
as noted, we probably need to find out why this `array_copy` exists, instead of just using `std::copy_n`.